kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/industrialforegoing/shaped.js

```diff
@@ -129,19 +129,25 @@ onEvent('recipes', (event) => {
         },
         {
             output: 'industrialforegoing:common_black_hole_tank',
-            pattern: ['AAA', 'A A', 'ABA'],
+            pattern: ['AAA', 'BCB', 'DED'],
             key: {
-                A: '#forge:sheetmetals/aluminum',
-                B: 'thermal:fluid_cell_frame'
```

Comment:
Would prefer to keep the fluid cell frame here, just for thematic reasons.

Reply:
my bad, copy pasting error 👍

Reply:
happens :D